apps/angular/src/app/features/anime/anime-management/anime-editing/anime-editing.component.html

```diff
@@ -0,0 +1,12 @@
+<a 
+  *ngIf="animeId$ | async as animeId" 
+  [routerLink]="['/anime', animeId]" 
+  mat-icon-button>
+    <mat-icon>arrow_back</mat-icon>
+</a>
+<camp-anime-form 
+  *ngIf="animeFormData$ | async as animeFormData" 
+  [animeFormData]="animeFormData" 
+  (submitEvent)="editAnime($event)">
+</camp-anime-form>
```

Comment:
Prefer to use self-closing tags for Angular components without child content:
```suggestion
<camp-anime-form 
  *ngIf="animeFormData$ | async as animeFormData" 
  [animeFormData]="animeFormData" 
  (submitEvent)="editAnime($event)"
/>
```